docs/source/additional/release.rst

```diff
@@ -10,11 +10,10 @@ These sets of versions have been tested for compatibility:
 
 .. compatibilitytable:: releases.yaml
 
-
 .. only:: html
 
-   .. note::
-      `JSON version of the release table </releases.json>`_
+    :download:`JSON version of the release table <../../../tmp/releases.json>`
```

Comment:
Are we sure the `tmp` folder is not deleted after build? 
Would it be safer to copy it somewhere more persistent?

Reply:
So you're saying that it works because of `:download:`, but it would be cleaner to find a way to point to a fixed url?

Reply:
@SdgJlbl:

Even though name might not be explicit, `tmp` is already where:
* The `releases.json` file is [stored](https://github.com/Substra/substra-documentation/blob/main/docs/source/conf.py#L29-L41)
* All the assets required for running the examples are [stored](https://github.com/Substra/substra-documentation/blob/main/docs/source/conf.py#L103-L138)  

---

@oleobal I get your point. For my personal culture, what automated tools do/might we have that rely on this file?